src/app/utils/encryptionUtils.ts

```diff
@@ -25,39 +38,37 @@ async function generateKeyArgon2(password: string, salt: string): Promise<string
 }
 
 export async function generatePasswordHash(password: string) {
-  const existingSalt = getSalt();
-  if (existingSalt) {
-    const argonHash = await generateKeyArgon2(password, existingSalt);
-    return {
-      salt: existingSalt,
-      hash: argonHash,
-    };
+  let salt = await ChromeStorage.getItem(SeedVaultStorageKeys.PASSWORD_SALT);
```

Comment:
It's a bit of a strange pattern that we are using the `SeedVaultStorageKeys` directly in the code instead of via the seed vault. Is there way to do something like `seedVault.getSalt()`?

Reply:
refactored the function so it will only deal with legacy so no need to support getting individual stored items anymore